testnets/nilliontestnet/assetlist.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../assetlist.schema.json",
```

Comment:
please use ../../assetlist.schema.json to point to the schema file.

Reply:
Done